src/components/header/header.jsx

```diff
@@ -1,9 +1,98 @@
 import './header.css'
-
+import { FiSearch } from 'react-icons/fi'
+import { HiOutlineChatBubbleLeftRight } from 'react-icons/hi2'
+import { IoNotificationsOutline } from 'react-icons/io5'
 const Header = () => {
+  const menu = ['Settings', 'Notification', 'Two Factor', 'Activity Status']
   return (
-    <div className="header">
-      <h1>React UI Components</h1>
+    <div className="header  d-flex align-items-center justify-content-center">
+      <nav className="navbarContainer d-flex justify-content-start align-items-center">
+        <div className="navbarBrand d-flex justify-content-center align-items-center">
+          <a href="/">
+            <svg
+              width="29"
+              height="32"
+              viewBox="0 0 29 32"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M5.24739 0C5.43789 1.34923 6.05445 2.59947 7.00463 3.56326C7.9548 4.52704 9.18739 5.15244 10.5176 5.34567C9.18701 5.53986 7.95432 6.16617 7.0042 7.13073C6.05408 8.0953 5.43769 9.34619 5.24739 10.696C5.05831 9.34953 4.44523 8.10115 3.49959 7.13697C2.55396 6.17279 1.32642 5.54446 0 5.34567C1.33018 5.15244 2.56276 4.52704 3.51294 3.56326C4.46311 2.59947 5.07967 1.34923 5.27018 0H5.24739Z"
```

Comment:
move svg into separate file, can be inside current folder or inside assets folder